service/project.go

```diff
@@ -245,6 +245,7 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 	id = projectRef.Id
 	origProjectRef := *projectRef
 
+	// TODO: should this use pointer booleans? otherwise the copy isn't quite right for repo projects
```

Comment:
Is this TODO still relevant? Did you want to do it as part of this PR?

Reply:
Right good question! I think I've decided it's not relevant, since this is the legacy UI and we aren't supporting it with the new structure. I'll remove the TODO.

Reply:
thanks!